queue/abstract/driver/fifo.lua

```diff
@@ -61,6 +61,11 @@ function tube.new(space, on_task_change)
     return self
 end
 
+-- method.grant grants provided user to all spaces of driver.
+function method.grant(self, user, opts, tp)
```

Comment:
Why do we need the `tp`-third arg if the method is called only with two args?

Reply:
I was designing this function like it was in `tube_grant_space` function in `queue/abstractl.lua` and thought that maybe this argument may be usable in future. But ok, deleted it.

Reply:
I got it, thank you.

It's usually better not to leave anything that might be useful in the future but isn't currently used. It just makes the code confusing.